src/app/component/app-footer/app-footer.component.html

```diff
@@ -8,7 +8,7 @@
           <ws-app-logo [classList]="'logo-dim app-logo'" [path]="'/page/home'" [logoSrc]="'/assets/instances/eagle/app_logos/KarmayogiBharat_Logo_Horizontal.svg'"></ws-app-logo>
         </div>
```

Comment:
please check for mweb as well, is it working fine?

Reply:
on mweb we don't have search box on top navbar